src/internal/connector/exchange/attachment.go

```diff
@@ -115,3 +155,37 @@ func attachmentWriter(ctx context.Context, service graph.Service, userID, folder
 
 	return uploadsession.NewWriter(messageID, url, size), nil
 }
+
+func attachmentEventWriter(ctx context.Context, service graph.Service, userID, calendarID, eventID string,
+	attachment models.Attachmentable, size int64,
+) (io.Writer, error) {
+	session := createuploadsession.NewCreateUploadSessionPostRequestBody()
```

Comment:
not sure if it really matters since it seems to compile, but technically events have their [own type](https://github.com/microsoftgraph/msgraph-sdk-go/blob/main/users/item/calendars/item/events/item/attachments/createuploadsession/create_upload_session_post_request_body.go#L17) for the upload session request body

Reply:
Library updated [ef219f0](https://github.com/alcionai/corso/pull/1345/commits/ef219f04ed7785f7a5bca671e8f99415db3d9a75)